VagrantProvision.sh

```diff
@@ -107,13 +118,13 @@ fi
 
 if ! ruby -v | grep --quiet 2.3.0; then
     # Ruby via rvm - from rvm.io
-    gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+    gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 2>&1
 
     curl -sSL https://raw.githubusercontent.com/rvm/rvm/master/binscripts/rvm-installer | bash -s stable
 
     source /home/vagrant/.rvm/scripts/rvm
 
-    rvm install ruby-2.3
+    stdbuf -o L -e L rvm install ruby-2.3
```

Comment:
This is the real magic one, `stdbuf` buffers the output of the command by line before actually writing to `stdout` or `stderr` saving us from the 200+ lines of single/blank characters in the ruby install process.